src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Passes/Apple_Wallet/Event_Data_Add_Info.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus\Passes\Apple_Wallet;
+
+/**
+ * Class Event_Data_Add_Info
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus\Passes\Apple_Wallet
+ */
+class Event_Data_Add_Info {
```

Comment:
This name feels weird, it says a lot without telling me much I guess that is where it lands.

`Event_Modifier`